VideoLocker/res/layout/row_myvideo_course_list.xml

```diff
@@ -4,41 +4,39 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content">
 
-    <LinearLayout
+    <RelativeLayout
         android:id="@+id/course_row_layout"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="@dimen/course_card_height"
         android:background="@color/white"
-        android:descendantFocusability="blocksDescendants"
-        android:orientation="vertical"
-        android:paddingBottom="11dp" >
+        android:descendantFocusability="blocksDescendants" >
 
-        <com.android.volley.toolbox.NetworkImageView
+        <ImageView
             android:id="@+id/course_image"
             android:layout_width="match_parent"
-            android:layout_height="120dp"
-            android:adjustViewBounds="true"
-            android:contentDescription="@string/course_image"
-            android:scaleType="centerCrop" />
+            android:layout_height="match_parent"
+            tools:background="@drawable/sample_course_image" />
 
         <RelativeLayout
             android:id="@+id/course_data"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_marginLeft="10dp"
-            android:layout_marginRight="10dp"
-            android:layout_marginTop="10dp" >
+            android:layout_height="@dimen/course_detail_card_height"
+            android:layout_alignParentBottom="true"
+            android:background="@color/transparent_white_85" >
 
             <org.edx.mobile.view.custom.ETextView
                 android:id="@+id/course_name"
                 style="@style/bold_grey_text"
                 android:layout_width="match_parent"
-                android:layout_height="match_parent"
-                android:layout_alignParentTop="true"
-                android:layout_marginBottom="10dp"
-                android:lineSpacingExtra="-1.5dp"
```

Comment:
I didn't see any need for it, the layout was looking fine without it, previously which wasn't the case, so, removed it.